frontend/src/layout/navigation-3000/navigationLogic.tsx

```diff
@@ -543,6 +551,7 @@ export const navigation3000Logic = kea<navigation3000LogicType>([
                             label: 'Session replay',
```

Comment:
Could've added `https://posthog.com/docs/web-analytics/revenue-tracking` for `Revenue Analytics`, I can add as a follow-up